extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -425,6 +445,35 @@ export class AmpStoryPage extends AMP.BaseElement {
     return Promise.all(mediaPromises);
   }
 
+  /**
+   * Finds embedded components in page and prepares them for their expanded view
+   * animation.
+   * @param {number} timeout
+   * @private
+   */
+  findAndPrepareEmbeddedComponents_(timeout) {
+    scopedQuerySelectorAll(this.element, EMBEDDED_COMPONENTS_SELECTORS)
+        .forEach(el => {
+          if (!el.classList.contains('i-amphtml-embedded-component')) {
+            let readyForResize = true;
+            listen(el, AmpEvents.SIZE_CHANGED, () => {
+              readyForResize = false;
+            });
+
+            this.timer_.poll(timeout, () => {
```

Comment:
I'm not sure I understand this.  Are you effectively trying to say "we will wait a maximum of `timeout` milliseconds after each resize event, before we consider the component to be at its final size"?

Reply:
Can we use the `debounce` helper? I feel like it communicates pretty clearly what it's going to do.